dashboard/lib/services/user/password_resetter_by_email.rb

```diff
@@ -45,6 +45,11 @@ def call
       private def send_reset_password_instructions
         raw = user.send(:set_reset_password_token)
         user.send(:send_devise_notification, :reset_password_instructions, raw, {to: email})
+        Cdo::Metrics.put(
+          'User', 'PasswordResetEmailSuccessful', 1, {
+            Environment: CDO.rack_env
+          }
+        )
```

Comment:
There is a clear pattern in the metrics logging that can be extracted into a separate method for better readability:
```ruby
private def log_user_metric(metric_name)
  Cdo::Metrics.put('User', metric_name, 1, Environment: CDO.rack_env)
end
```
```suggestion
        log_user_metric('PasswordResetEmailSuccessful')
```